examples/nft/assets/nft_approval.py

```diff
@@ -9,12 +9,12 @@ def approval_program():
     hash is an hash of the external resources data.
 
     Commands:
-      create    Creates a new NFT. Expects 2 additional arguments:
+    create    Creates a new NFT. Expects 2 additional arguments:
```

Comment:
can you please check the indentation here? It seems wrong

Reply:
we may add a tab here, i think this is about preference only